proto/gz/msgs/dynamic_detachable_joint.proto

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2025 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Contributor: Adarsh Karan K P
+// Neobotix GmbH
+
+syntax = "proto3";
+
+package gz.msgs;
+
+option java_package = "com.gz.msgs";
+option java_outer_classname = "DynamicDetachableJointProtos";
```

Comment:
If we move the `AttachDetachResponse` to `Result`, this will just be `AttachDetachRequest`